src/app/pages/details/details.js

```diff
@@ -67,18 +67,21 @@ export default class Details extends BaseView {
     toggleFixedClass() {
         let floatingMenu = document.querySelector('.floating-menu>.box');
 
+        var body = document.body,
```

Comment:
`let` instead of `var`. It doesn't make things run differently, but when I read it, it makes me think there's something unconventional going on.